src/modules/rlm_mschap/auth_wbclient.c

```diff
@@ -28,6 +28,7 @@ RCSID("$Id$")
 #include <freeradius-devel/rad_assert.h>
 
 #include <wbclient.h>
+#include <core/ntstatus.h>
```

Comment:
Comes as part of Samba, so if wbclient.h is there (tested by configure script), core/ntstatus.h must be as well.